src/pages/StatisticsChart/monetary/TotalSupply.tsx

```diff
@@ -192,9 +192,9 @@ const toCSV = (statisticTotalSupplies: ChartItem.TotalSupply[]) =>
   statisticTotalSupplies
     ? statisticTotalSupplies.map(data => [
         data.createdAtUnixtimestamp,
-        shannonToCkbDecimal(data.circulatingSupply, 8),
-        shannonToCkbDecimal(data.lockedCapacity, 8),
-        shannonToCkbDecimal(data.burnt, 8),
+        shannonToCkbDecimal(data.circulatingSupply, 8).toString(),
+        shannonToCkbDecimal(data.lockedCapacity, 8).toString(),
+        shannonToCkbDecimal(data.burnt, 8).toString(),
```

Comment:
What is the purpose of converting number to string? I looked at the code and it seems to have no effect.

It seems to be an alignment with other BigNumbers. If that's the case, consider changing the signature of `dataToCsv` from `(data?: (string | number)[][])` to `(data?: string[][])` at the source.

Reply:
> What is the purpose of converting number to string? I looked at the code and it seems to have no effect.
> 
> It seems to be an alignment with other BigNumbers. If that's the case, consider changing the signature of `dataToCsv` from `(data?: (string | number)[][])` to `(data?: string[][])` at the source.

A mistake I have made, these changes should be reverted, I think I have done this before pushing the commit, but actually not, I will remove this.

Reply:
done